src/Storage/Repository/PgInstanceRepository.cs

```diff
@@ -295,7 +295,7 @@ await using (NpgsqlDataReader reader = await pgcom.ExecuteReaderAsync())
         }
 
         /// <inheritdoc/>
-        public async Task<(Instance Instance, long InternalId)> GetOne(int instanceOwnerPartyId, Guid instanceGuid, bool includeElements = true)
+        public async Task<(Instance Instance, long InternalId)> GetOne(Guid instanceGuid, bool includeElements = true)
```

Comment:
Default values can be useful, but completely unnecessary in this case.

Reply:
I get compile errors if I also remove the default value from the interface. I don't like default values in general. It's better to update call points. There aren't that many of them.

Reply:
The must interesting thing to address related to this is how to maintain the two instance properties lastchanged and readstatus. In many use cases the data elements are fetched only to make sure that lastchanged and readstatus are set the same way as with the Cosmos repo. The lastchanged semantics is not obvious - the value in the json doc in the db is not the value used externally. One solution could be to always update the two properties directly on the instance for every data element write operation. Then the semantics of the includeDataelements parameter on GetOne would be obvious.